.github/workflows/rust.yml

```
@@ -6,130 +6,74 @@ on:
         branches:
             - main
 
+env:
+  CARGO_INCREMENTAL: 0
+
 jobs:
     lint:
         name: "Lint"
         runs-on: ubuntu-latest
+        container:
+          image: ghcr.io/brewingbytes/tauri-builder
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)